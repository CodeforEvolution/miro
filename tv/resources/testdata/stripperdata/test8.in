<p>
  Q: What did the <a href="http://en.wikipedia.org/wiki/Snail">snail</a> say 
  while riding on the <a href="http://en.wikipedia.org/wiki/Turtle">turtle</a>'s back?
</p>
<p>
  A: Whee!
</p>
