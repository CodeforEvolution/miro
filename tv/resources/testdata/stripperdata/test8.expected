(u"Q: What did the snail say while riding on the turtle's back?\nA: Whee!", [(16, 21, u'http://en.wikipedia.org/wiki/Snail'), (46, 52, u'http://en.wikipedia.org/wiki/Turtle')])
